src/pages/news/2021-06-24-roadmap.en.md

```diff
@@ -0,0 +1,32 @@
+---
+path: "/en/news/2021-06-24-roadmap/"
+date: "2021-06-24"
+title: "Roadmap Signalen July 2021 - December 2021"
+lang: en
+template: default
+---
+
+## Roadmap Signalen July 2021 - December 2021
+The product steering group has been busy working with all community members of Signalen to
+determine which functional improvements they would like to see within Signalen.
+
+The second half year we have decided to focus our efforts on making Signalen more user-friendly for the people who handle the notifications. Below is a summary of the features we will be working on:
+
+---
+
+| Priority | Feature theme (epic) | Description | Complexity/ Scope of work |
+| -------- | -------- | -------- | -------- |
+| 1 | Granularity of read and write permissions | Adjust dropdown to read-only users -> small (depends on 1) | medium / large |
+| 2a | Filtering | Refactor | large |
+| 2b | Filtering | Autocomplete add address | small |
+| 3a | More user-friendly management | Routing rules in Django admin | small |
+| 3b | More user-friendly management | Machine learning, research needed | large |
+| 3c | More user-friendly management | Additional questions implementer in front-end| large |
+| 4 | Enrich data dump | Enrich data dump| small/medium |
+| 4 | Notify practitioners| Send mail / Make configurable | medium |
+
+---
+<br/>
```

Comment:
HTML in news section?

Reply:
yes so the table is not added to the text. This is allowed in this setup